AudioWave/AudioPlayer/AudioSource.swift

```diff
@@ -77,13 +76,13 @@ class AudioSource: NSObject
     //MARK: - Titles
     func track() -> String {
         //return mediaItem?.title ?? "No Track Name"
-        guard let mediaItem = playlist?.currentItem() else { return "FIXME track()" }
+        let mediaItem = playlistController!.currentItem
```

Comment:
the app crashes here when i go to play an item. 
Error is: Unexpectedly found nil when unwrapping an optional.

![Screen Shot 2021-01-08 at 3 49 49 PM](https://user-images.githubusercontent.com/611873/104057811-25d62680-51c9-11eb-9650-e6ee7f1b222b.png)